api/urls.py

```diff
@@ -130,6 +131,19 @@ def to_url(self, value):
         ),
     ]
 
+
+if settings.PHONES_ENABLED and settings.IQ_ENABLED:
+    from .views.phones import inbound_sms_iq
+
+    urlpatterns += [
+        path(
+            "v1/inbound_sms_iq/",
+            enable_if_setting("IQ_ENABLED")(inbound_sms_iq),
```

Comment:
This is a little weird - the URL is enabled if `settings.IQ_ENABLED`. But, if later it goes False, then it is disabled. Which will only happen in a test.

I've got ideas for making this less weird, but that's future code.